Dockerfile

```diff
@@ -15,6 +15,7 @@ RUN cargo chef cook --release --recipe-path recipe.json
 
 FROM node:16-alpine as node_builder
 ADD --chown=node:node ./static /siwe-oidc/static
+ADD --chown=node:node ./js/ui/img /siwe-oidc/static/img
```

Comment:
Why not put the images under `./static` directly? BTW the weird structure is because I couldn't figure out how to move the static files in the build directory with webpack.

Reply:
It was performing some path checks and when in the static folder it wouldn't build, this was the only way I made it work that's why :(